tests/erc190/test_package.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from erc190 import Package
+from erc190.exceptions import ValidationError
+from jsonschema import ValidationError as VE2
+
+
+def test_ethpm_exists():
+    assert Package
+
+
+@pytest.fixture()
+def valid_package():
+    return "./tests/erc190/validSample.json"
+
+
+@pytest.fixture()
+def invalid_package():
+    return "./tests/erc190/invalidSample.json"
+
+
+def test_ethpm_instantiates_with_valid_package(valid_package):
+    current_package = Package(valid_package)
+    assert current_package.lockfile == valid_package
+    assert current_package.parsed_json
+
+
+def test_ethpm_doesnt_instantiate_with_invalid_package(invalid_package):
+    with pytest.raises(VE2):
```

Comment:
I think it would be better if all validation errors resulted in an `erc190.exceptions.ValidationError` so that people only have to watch for one error type during validation.